nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -403,7 +403,8 @@ def transcribe(
         """
         Uses greedy decoding to transcribe audio files. Use this method for debugging and prototyping.
         Args:
-            audio: (a list) of paths to audio files. \
+            audio: (a single or list) of paths to audio files or a np.ndarray audio array.
+                Can also be a dataloader object that provides values that can be consumed by the model.
```

Comment:
Add torch.dataloader to `audio` types in the func signature.

Reply:
Done